code/game/objects/machinery/nuclearbomb.dm

```diff
@@ -65,12 +65,16 @@
 	update_minimap_icon()
 	// The timer is needed for when the signal is sent
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_NUKE_START, src)
+	log_game("[usr] has enabled the nuke at [AREACOORD(src)]")
```

Comment:
do not use usr, it is a very innacurate byond var that should only be used in a few niche cases. pass the user to the proc as an argument properly instead

Reply:
doen